src/data/localization.rs

```diff
@@ -700,10 +696,9 @@ impl FileHandler<(Language, Vec<LocaEntry>)> for Localization {
                 if check.iter().any(|t| t.is(lang_str))
                     || (check.is_empty() && skip.iter().all(|t| !t.is(lang_str)))
                 {
-                    langs.set(lang.to_idx(), true);
+                    self.check_langs[lang.to_idx()] = true;
```

Comment:
Note that there's a bug here which could affect the benchmarks. `check_langs` is initialized to all true. Iff there's a languages block, then it should only contain the languages configured there. So the `= true` is a no-op here because you removed the initialization to zero in this function.

So you may have been measuring mods that due to this bug are scanning 11x the usual number of languages.